src/messages/javascript.js

```diff
@@ -239,3 +239,30 @@ export const KNOWN_LIBRARY = {
     'testcases_content', 'test_packed_packages', 'blacklisted_js_library',
   ],
 };
+
+function deprecatedAPI(api) {
+  return {
+    code: apiToMessage(api),
+    legacyCode: [
+      'js',
+      'deprecated',
+      api,
+    ],
+    message: _(`"${api}" is deprecated or unimplemented`),
+    description: _(singleLineString`APIs that have been deprecated by Chrome
+      have not been implemented by Firefox.`),
+  };
+}
+
+export const TABS_GETSELECTED = deprecatedAPI('tabs.getSelected');
+export const TABS_SENDREQUEST = deprecatedAPI('tabs.sendRequest');
+export const TABS_GETALLINWINDOW = deprecatedAPI('tabs.getAllInWindow');
+export const TABS_ONSELECTIONCHANGED = deprecatedAPI(
+  'tabs.onSelectionChanged');
+export const TABS_ONACTIVECHANGED = deprecatedAPI('tabs.onActiveChanged');
+export const EXTENSION_SENDREQUEST = deprecatedAPI('extension.sendRequest');
+// Sadly we'll have to concat this one to fit under 26 chars.
+export const EXTENSION_ONREQUESTEXTERN = deprecatedAPI(
```

Comment:
I would concat the group (`EXT` over `EXTENSION`) rather than the specific message. eg `EXT_ONREQUESTETERNAL`